hellosign_docsignature_template.go

```diff
@@ -129,6 +129,37 @@ func (m *Client) GetEmbeddedTemplateEditURLForPreview(templateID string) (*model
 	return data.GetEmbedded(), nil
 }
 
+// GetEmbeddedTemplateEditURLForEdit - Retrieves an embedded template object for edit.
+func (m *Client) GetEmbeddedTemplateEditURLForEdit(templateID string, customFields string) (*model.EmbeddedTemplateEditURL, error) {
```

Comment:
I think we should modify the `GetEmbeddedTemplateEditURL` instead as that was the original method intended for edit.

HelloSign changed the implementation and hence the existing code is not functional.

Also `GetEmbeddedTemplateEditURL` is not used in the code so should be safe to be removed.